src/server/frontend_xwayland/xwayland_server.cpp

```diff
@@ -117,7 +117,7 @@ auto fork_xwayland_process(
 
     case 0:
         exec_xwayland(spawner, xwayland_path, wayland_pipe.client, x11_wm_pipe.server);
-        perror("Failed to start XWayland, should be unreachable");
+        mir::log_error("Cannot execute Xwayland: --xwayland-path %s", xwayland_path.c_str());
```

Comment:
Is using `log_error()` in another process a good idea? Even if so, I still want the errno message to be printed. Alternative here: fcf5af1a6329d9d460cbc71360a66e64fcf8a2b5